### DISCUSSION TITLE
'findLocale' does not exist in type LanguageDetectorOption

### DISCUSSION THREAD

**Discussion by eliawk** [sergiodxa/remix-i18next#215] [Q&A] [answered]
If I use the `findLocale` function in the detection
```
export let i18n = new RemixI18Next({
  detection: {
    supportedLanguages: ["es", "en"],
    fallbackLanguage: "en",
    async findLocale(request) {
      let locale = request.url.pathname.split("/").at(1);
      return locale;
    },
  },
});
```

I get this error:
Object literal may only specify known properties, and 'findLocale' does not exist in type 'LanguageDetectorOption'.ts(2353)
server.d.ts(67, 5): The expected type comes from property 'detection' which is declared here on type 'RemixI18NextOption'

**Comment by sergiodxa** [comment, marked as answer]
Are you on the latest version?

You can see an example usage here where TS is not complaining https://github.com/sergiodxa/remix-i18next/blob/8e800d55f344fef925f44543d4d83d2f1265d069/src/server.test.ts#L101-L119

And if you check the code published to npm the `.d.ts` files includes `findLocale` https://www.npmjs.com/package/remix-i18next?activeTab=code
